magenta/models/music_vae/js/index.ts

```diff
@@ -385,6 +349,9 @@ export {
 	bitsToInts,
 	intsToOneHot
 }
+
+export const isDeviceSupported:boolean = isWebGLSupported() && !isSafari();
```

Comment:
no need to type `boolean` this is inferred by the compiler

Reply:
Done